packages/alpha/src/__tests__/api/alertsApi.int.spec.ts

```diff
@@ -139,4 +139,13 @@ describe('alerts api', () => {
     ]);
     expect(response).toEqual({});
   });
+
+  itif(client)('delete channel', async () => {
+    const response = await client!.alerts.deleteChannels([
+      {
+        externalId: channelExternalId,
+      },
+    ]);
+    expect(response).toEqual({});
```

Comment:
> Check for status 200?
we technically don't have access to the status at this level anymore (at least the easy way), if 400+ status happens it will throw an error at earlier stages